ladybug/futil.py

```diff
@@ -146,32 +167,45 @@ def copy_files_to_folder(files, target_folder, overwrite=True):
     return [os.path.join(target_folder, os.path.split(f)[-1]) for f in files]
 
 
-def bat_to_sh(file_path):
-    """Convert honeybee .bat file to .sh file.
+def copy_file_tree(source_folder, dest_folder, overwrite=True):
```

Comment:
Couldn't we just use `distutils.dir_util.copy_tree` for this?

https://docs.python.org/2/distutils/apiref.html#distutils.dir_util.copy_tree

Reply:
You are right. For some reason, I thought that `dir_util.copy_tree` had issues in IronPython but I just tested it and it worked. Maybe it was just an issue with Rhino 5's IronPython. Ok. I'll just remove it from this PR>